Resources/config/web_push.yml

```diff
@@ -3,7 +3,7 @@ parameters:
 
 services:
   minishlink_web_push:
-    class: %minishlink_web_push.class%
-    arguments: [%minishlink_web_push.api_keys%, %minishlink_web_push.default_options%, %minishlink_web_push.timeout%, null]
+    class: '%minishlink_web_push.class%'
+    arguments: ['%minishlink_web_push.api_keys%', '%minishlink_web_push.default_options%', '%minishlink_web_push.timeout%', null]
```

Comment:
api_keys and default_options are arrays, not scalars, they shouldn't be wrapped with quotes I guess.